source/utils/RandomUtils.cc

```diff
@@ -0,0 +1,42 @@
+// ----------------------------------------------------------------------------
+// nexus | RandomUtils.cc
+//
+// This class defines commonly used functions to generate random quantities.
+//
+// The NEXT Collaboration
+// ----------------------------------------------------------------------------
+#include "RandomUtils.h"
+
+#include <Randomize.hh>
+
+#include "CLHEP/Units/SystemOfUnits.h"
+
+namespace nexus {
+
+  G4double UniformRandomInRange(G4double max_value, G4double min_value)
+  {
+    if (max_value == min_value){
+      return min_value;
+    } else {
+      return (G4UniformRand()*(max_value - min_value) + min_value);
+    }
+  }
+
+
+  G4ThreeVector Direction(G4double costheta_min, G4double costheta_max,
```

Comment:
`Direction` is not a very descriptive name. Why not `UniformRandomDirection` or something similar?

Reply:
What about `RandomDirection`?

Reply:
Actually, it should be `RandomDirectionInRange`.

Reply:
Yes, that's fine too.